docs/content_management/taxonomy/taxonomy_api.md

```diff
@@ -17,6 +17,15 @@ and use `TaxonomyServiceInterface::loadEntryByIdentifier()`:
 [[= include_file('code_samples/api/public_php_api/src/Command/TaxonomyCommand.php', 43, 46) =]]
 ```
 
+!!! note
+
+    A taxonomy entry identifier is unique per taxonomy. If you have [several taxonomies](taxonomy.md#customize-taxonomy-structure), you can increase code readability by always passing the taxonomy identifier even when it's the default one.
```

Comment:
It should be specified what the default taxonomy is or better reference the method returning the default taxonomy.

Reply:
@ms2ad
```suggestion
    A taxonomy entry identifier is unique per taxonomy. If you have [several taxonomies](taxonomy.md#customize-taxonomy-structure), you can increase code readability by always passing the taxonomy identifier even when it's the default one. The default taxonomy is `tags` if it exists, else the first configured taxonomy (see `\Ibexa\Taxonomy\Service\TaxonomyConfiguration::getDefaultTaxonomyName` for details).
```